protocol/cmd/dydxprotocold/cmd/root.go

```diff
@@ -119,7 +127,28 @@ func NewRootCmdWithInterceptors(
 				return err
 			}
 
-			serverCtxInterceptor(server.GetServerContextFromCmd(cmd))
+			serverCtx := server.GetServerContextFromCmd(cmd)
+			if ddErrorTrackingFormatterEnabled :=
+				serverCtx.Viper.Get(protocolflags.DdErrorTrackingFormat); ddErrorTrackingFormatterEnabled != nil {
+				if enabled, err := cast.ToBoolE(ddErrorTrackingFormatterEnabled); err == nil && enabled {
+					// Error fields are default set under `error`
+					// Extract + add the kind and message field
+					zerolog.ErrorMarshalFunc = func(err error) interface{} {
```

Comment:
can we use a sync.Once global to ensure that this is only ever done once otherwise any tests that launch multiple instances of this app in parallel will hit a data race.

Reply:
done